central/notifiers/utils/encryption.go

```diff
@@ -158,6 +158,11 @@ func getCredentials(notifier *storage.Notifier) (string, error) {
 		return notifier.GetPagerduty().GetApiKey(), nil
 	case pkgNotifiers.GenericType:
 		return notifier.GetGeneric().GetPassword(), nil
+	case pkgNotifiers.MicrosoftSentinelType:
+		if notifier.GetMicrosoftSentinel().GetSecret() != "" {
+			return notifier.GetMicrosoftSentinel().GetSecret(), nil
+		}
+		return notifier.GetMicrosoftSentinel().GetClientCertAuthConfig().GetPrivateKey(), nil
```

Comment:
Is `PrivateKey` the only thing that we want to encrypt? Or should `ClientCert` be encrypted too? In other words, can someone do any damage if they got hands on `ClientCert`?

Reply:
No, clientCert is the public and will be presented to all servers the notifier tries to connect to.